meta-mel/recipes-core/os-release/os-release.bbappend

```diff
@@ -0,0 +1,13 @@
+OS_RELEASE_FIELDS = "PRETTY_NAME NAME VERSION_ID VERSION VERSION_CODENAME ID HOME_URL SUPPORT_URL BUG_REPORT_URL BUILD_ID"
+OS_RELEASE_UNQUOTED_FIELDS = "ID VERSION_ID VARIANT_ID"
+
+ID = "${DISTRO}"
+NAME = "${OS_NAME}"
+VERSION = "${DISTRO_VERSION}${@' (%s)' % DISTRO_CODENAME if 'DISTRO_CODENAME' in d else ''} ${YOCTO_RELEASE}"
+VERSION_ID = "${DISTRO_VERSION}"
+VERSION_CODENAME = "${YOCTO_RELEASE}"
+PRETTY_NAME = "${DISTRO_NAME} ${VERSION}"
+HOME_URL = "https://www.mentor.com/embedded-software/linux/mel-flex-os/"
+SUPPORT_URL = "https://www.mentor.com/support"
+BUG_REPORT_URL = "https://www.mentor.com/support"
```

Comment:
These should be set in mel.conf, either globally or with _pn-os-release. Actually there's an argument to be made that they should all be there if needed. If they're kept here, they need _mel at a minimum. Still, it's clear the urls don't belong here.

Reply:
Sure I will move the urls to mel.conf